models/staging/stg_workday__personal_information.sql

```diff
@@ -60,7 +60,7 @@ final as (
         tobacco_use as is_tobacco_use,
         type
     from fields
-    where coalesce(_fivetran_active, true)
+    where {{ dbt.current_timestamp_backcompat() }} between _fivetran_start and _fivetran_end
```

Comment:
Do you know if we still need to use `dbt.current_timestamp_backcompat` or are we able to swap to just using `dbt.current_timestamp`? 

I am unsure what the ramifications of this swap would be. I know we use backcompat in the other packages, but I am wondering if this was only to ensure backwards compatibility at the time of the upgrades. I wonder if this is still needed. Can you look into this and confirm?

Reply:
Hmm good point, it does [say](https://github.com/dbt-labs/dbt-core/blob/v1.7.8/core/dbt/include/global_project/macros/adapters/timestamps.sql) that the backcompat is deprecated. I switched it over and it looks fine!
 

![image](https://github.com/fivetran/dbt_workday/assets/91097070/25ab1854-5c27-4beb-b076-72fb4278315c)